roofit/xroofit/src/xRooNode.cxx

```diff
@@ -7717,7 +7911,7 @@ std::vector<double> xRooNode::GetBinErrors(int binStart, int binEnd, const xRooN
                                                                          : fitResult().fComp);
    }
 
-   if (!fr->_finalPars) {
+   if (!GETDMP(fr.get(),_finalPars)) {
```

Comment:
```suggestion
      if (fr->floatParsFinal().empty()) {
```
You can do this after our recent change to always initialize `_finalPars`, right?